namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-frontend-staging/resources/route53.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_route53_zone" "cla_frontend_route53_zone" {
+  name = "cases.civillegaladvice.service.gov.uk"
+
+  tags = {
+    team_name              = var.team_name
+    business-unit          = var.business-unit
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "cla_frontend_route53_zone_sec" {
+  metadata {
+    name      = "cla-frontend-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.cla_frontend_route53_zone.zone_id
+  }
+}
+
+resource "aws_route53_record" "add_a_record" {
+  name    = "cases.civillegaladvice.service.gov.uk"
```

Comment:
name = "." ?

Reply:
Sorry not sure what you mean. You mean to remove the spacing?

Reply:
I mean, the record name is just "." in the zone, right? otherwise it will be cases.civillegaladvice.service.gov.uk.cases.civillegaladvice.service.gov.uk

Reply:
ahhh cool ty